std_msgs/Header header
CommandID command
ObjectOfInterestPosition object
